visits-service-new/src/test/java/com/petclinic/visits/visitsservicenew/Exceptions/BadRequestExceptionTest.java

```diff
@@ -0,0 +1,39 @@
+package com.petclinic.visits.visitsservicenew.Exceptions;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
```

Comment:
Could you explain the reason for adding the `@Extendwith` annotation?

Reply:
Was basing it off of old code, have removed and tested it and it works without it too!